ui/src/redux/create.js

```diff
@@ -16,7 +16,7 @@ export default function createStore(llClient, router, data) {
   ];
 
   let finalCreateStore;
-  if (__DEVELOPMENT__ && __CLIENT__ && __DEVTOOLS__) {
+  if (__DEVELOPMENT__ && __CLIENT__ && (__DEVTOOLS__ || window.devToolsExtension)) {
```

Comment:
@ht2 can you check this line, I think you might have had an issue with this at some point.